acceptance/go.mod

```diff
@@ -12,3 +12,61 @@ require (
 	k8s.io/apimachinery v0.19.3
 	k8s.io/client-go v0.19.3
 )
+
+require (
```

Comment:
Could we merge these require blocks?

Reply:
go 1.17 groups the require blocks as indirect and direct, and since this is generated I think we should leave it as it was generated via `go mod tidy`

Reply:
Thanks for explaining!